packages/utils/src/abi_encoder/evm_data_types/address.ts

```diff
@@ -0,0 +1,51 @@
+/* tslint:disable prefer-function-over-method */
+import { DataItem } from 'ethereum-types';
+import * as ethUtil from 'ethereumjs-util';
+import * as _ from 'lodash';
+
+import { AbstractDataTypes, DataTypeFactory } from '../abstract_data_types';
```

Comment:
I think importing from a folder is somewhat confusing, since it isn't immediately obvious which file the import is actually located in. Ideally we would import directly from the relevant files OR we can import/export the relevant files in each `index.ts` and then have all of the top level imports come from there.